### PR TITLE
[Molgenis-logging][FEAT] Add helm config for logging Molgenis instances

### PR DESCRIPTION
**Pull request by christianhilbrands**
Feat: Added configuration for elasticsearch.
Todo: test configuration, set hardcoded values to one place and reference them, add Kibana support

### CODE REVIEW

File: charts/molgenis-logging/values.yaml.bu [hunk 1]
```diff
@@ -0,0 +1,27 @@
+clusterName: "edge"
```
Comment (sidohaakma) [inline charts/molgenis-logging/values.yaml.bu]: Is this file obsolete?

File: charts/molgenis-logging/values.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    }
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+    rules:
+      - host: "logging.edge.molgenis.org"
+        http:
+          - path: /
+            backend:
+              serviceName: "elasticsearch"
+              servicePort: 9200
```
Comment (sidohaakma) [inline charts/molgenis-logging/values.yaml]: I am missing Kibana configuration regarding the UI to read the logging.

File: charts/molgenis-logging/values.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  esJavaOpts: "-Xmx2g -Xms2g"
+
+  # Enabling this will publically expose your Elasticsearch instance.
+  # Only enable this if you have security enabled on your cluster
+  ingress:
+    enabled: true
+    hosts:
+      - logging.edge.molgenis.org
+    annotations: {
+      ingress.kubernetes.io/whitelist-source-range: 195.169.22.0/23
```
Comment (sidohaakma) [inline charts/molgenis-logging/values.yaml]: I think the whitelist-source-range mapping is not working. When looked at the molgenis118 I could not get through. When you setup the stack on the cluster please take a look if you will.

File: charts/molgenis-logging/requirements.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+dependencies:
```
Comment (sidohaakma) [inline charts/molgenis-logging/requirements.yaml]: Nice!

File: charts/molgenis-logging/requirements.lock [hunk 5]
```diff
@@ -0,0 +1,9 @@
+dependencies:
```
Comment (sidohaakma) [inline charts/molgenis-logging/requirements.lock]: Nice!

File: charts/molgenis-logging/values.yaml (hunk 2 above)
Comment (christianhilbrands) [reply charts/molgenis-logging/values.yaml]: Added in an later commit.

File: charts/molgenis-logging/values.yaml.bu (hunk 1 above)
Comment (christianhilbrands) [reply charts/molgenis-logging/values.yaml.bu]: No, was an error in my brain. My apologies ;-)

File: charts/molgenis-logging/values.yaml (hunk 3 above)
Comment (christianhilbrands) [reply charts/molgenis-logging/values.yaml]: corrected it in an later pull request.

File: charts/molgenis-logging/values.yaml.bu (hunk 1 above)
Comment (sidohaakma) [inline charts/molgenis-logging/values.yaml.bu]: Remove this file from PR.

File: charts/molgenis-logging/values.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        secretKeyRef:
+          name: "elastic-credentials"
+          key: logging.password
+    - name: ELASTIC_USERNAME
+      valueFrom:
+        secretKeyRef:
+          name: "elastic-credentials"
+          key: logging.username
+kibana:
+  image: "docker.elastic.co/kibana/kibana-oss"
```
Comment (sidohaakma) [inline charts/molgenis-logging/values.yaml]: This is really nice. The oss version!

File: charts/molgenis-logging/Chart.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+appVersion: "1.0"
```
Comment (sidohaakma) [inline charts/molgenis-logging/Chart.yaml]: appVersion is the version of the reference app so in this case: elasticsearch and kibana *7.2.0*

File: charts/molgenis-logging/values.yaml [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+          name: "elastic-credentials"
+          key: logging.username
+    - name: 'ELASTICSEARCH_PASSWORD'
+      valueFrom:
+        secretKeyRef:
+          name: "elastic-credentials"
+          key: logging.password
+  elastic:
+    username: xxxx
+    password: xxxx
```
Comment (sidohaakma) [inline charts/molgenis-logging/values.yaml]: Maybe add this to enable authentication on Kibana:

```
kibanaConfig:
  kibana.yml: |
    xpack.security.encryptionKey: from_second_secret (at least 32 characters)
```

This way we enable authentication on Kibana. We then can configure OpenID connect or something like that?

**Comment by sidohaakma** [comment]
Please cleanup the commit history ;-D.

File: charts/molgenis-logging/values.yaml.bu (hunk 1 above)
Comment (christianhilbrands) [reply charts/molgenis-logging/values.yaml.bu]: Was already removed. Not found.

File: charts/molgenis-logging/Chart.yaml (hunk 7 above)
Comment (christianhilbrands) [reply charts/molgenis-logging/Chart.yaml]: Thanks for the explanation. I have changed it.

**Pull request closed without merging**